homeassistant/components/google_assistant/trait.py

```diff
@@ -1210,6 +1225,28 @@ def query_attributes(self):
     async def execute(self, command, data, params, challenge):
         """Execute an SetModes command."""
         settings = params.get("updateModeSettings")
+
+        if self.state.domain == input_select.DOMAIN:
+            option = params["updateModeSettings"]["option"]
+            await self.hass.services.async_call(
+                input_select.DOMAIN,
+                input_select.SERVICE_SELECT_OPTION,
+                {
+                    ATTR_ENTITY_ID: self.state.entity_id,
+                    input_select.ATTR_OPTION: option,
+                },
+                blocking=True,
+                context=data.context,
+            )
+
+            return
+        if self.state.domain != media_player.DOMAIN:
```

Comment:
I would have preferred same style else if here. But I suppose can be fixed later.

Reply:
Our linter will complain if you use `elif` here because of the `return` in the last block.